python/sandbox/das_gate/test_das.metta

```diff
@@ -1,6 +1,7 @@
 !(extend-py! dasgate)
 
-!(bind! &das (new-das))
+;!(bind! &das (new-das))
+!(bind! &das (new-remote-das 104.238.183.115 8081))
```

Comment:
I would put such an IP in the open repo

Reply:
It can be either a function for getting IP from env (by wrapping Python `os.environ`), or just
`!(bind! &das (new-remote-das (Error "Put your IP here" ()) 8081))`